### ISSUE TITLE
Факторы, влияющие на дозировку и существующее ПО

### ISSUE THREAD

**Issue by pivovarov-mai** [pivovarov-mai/onmp#35]

**Comment by VanillaFox** [comment]
# Дозировки препаратов
Дозировка лекарственного препарата зависит от нескольких параметров: индивидуальные физиологические параметры пациента, форма препарата, первое/повторное применение.

## Физиологические параметры пациента
Дозировка считается на основе одного из 3 параметров (в зависимости от препарата):
+ кому мы даем препарат: `ребенку или взрослому`
+ `масса` пациента
+ `возраст` пациента

Корреляция дозировки от одного из этих параметров указывается в инструкции по применению.

## Формы препратара
Один препарат может быть в разных формах: порошок, таблетки, капсулы, капли, сироп и прочее. В зависимости от `формы препарата` также меняется дозировка. Эту информацию тоже можно получить из иснтрукции по применению препарата. 

## Первое/повторное применение
Иногда бывает такое, что врач может повторно назначить курс. Но для некоторых препаратов `повторное применение` должно быть с другими дозировками. Если такое имеет место быть, то эта информация указывается в отдельном пункте "повторное применение" в инструкции по применению.

## Противопоказания
Препараты могут иметь ряд противопоказаний. Применение некоторых лекарств недопустимо при определенных диагнозах, например: сердечная недостаточность, беременность.
Также возможен вариант с аллергической реакцией человека на вещество состава. Например, препарат *Гадобутрол-ТЛ* нельзя применять, если у человека гиперчувствительность к *гадобутролу*, который является действующим веществом.

Информацию по препаратам (их формам, дозировкам, противопоказаниям и прочим характеристикам) я буду брать из [Регистра лекарственных средств России](https://www.rlsnet.ru/).

# Аналоги
Существует два типа калькуляторов дозировок:
+ калькуляторы на основе готовых формул, например: правило Янга, правило Кларка,дозис фактор. То есть калькуляторы, основываясь на общих формулах, выдают дозировку, не привязываясь к конкретному препратару. Примером такого калькулятора может служить [этот калькулятор расчета доз лекарств для детей](https://medical-club.net/onlajn-raschet-doz-lekarstv-dlja-detej/).
+ калькуляторы, которые делают расчет на основе выбранного пользователем препарата. Примером такого калькулятора служит Infinite Dose - приложение, доступное на Android.

В нашем проекте я буду реализовывать второй тип калькуляторов. На мой взгляд, у нас достаточно официальной информации по препаратам, чтобы использовать ее для получения дозировок без потенциальной погрешности, которую могут дать общие формулы.

**Comment by pivovarov-mai** [comment]
> Корреляция дозировки от одного из этих параметров указывается в инструкции по применению.

А может все-таки от каждого из параметров?)

> * `возраст` пациента.

а разве по возрасту нельзя определить ребенок/взрослый?

**Comment by VanillaFox** [comment]
нет, конкретно один из параметров. я это уточняла у Ильвиры Марии. она сказала "при дозировке учитывается либо ребенок/взрослый, либо возраст пациента, либо вес"

по возрасту можно определить, конечно. но тут имелось в виду то, что дозировка либо рассчитывается из возраста (1 капля на год жизни), либо берется из схемы применения. я это указала в скобках в качестве примера

**Comment by pivovarov-mai** [comment]
Статус на ревью предполагает, что мы хотим что-то добавить в репозиторий и необходимо сделать ревью, чтобы одобрить запрос. В данном случае кроме обсуждени ятикета ничего не происходит, поэтому возвращаю статус "В работе".

**Comment by VanillaFox** [comment]
Хорошо
А что ещё можно добавить к этой теме?
Мне казалось, что я указала все пункты, которые мы обсуждали с Марией
